nca/FWRules/ConnectivityGraph.py

```diff
@@ -91,6 +100,36 @@ def _get_peer_color(is_livesim, is_ip_block):
             return "red2"
         return "blue"
 
+    def get_connections_without_fw_rules_txt_format(self):
+        """
+        :rtype: str
+        :return: a string of the original peers connectivity graph content (without adding fw-rules or livesim peers)
+        """
+        lines = set()
+        workload_name_to_peers_map = {}  # a dict from workload_name to pods set, to track replicas and copies
+        for connections, peer_pairs in self.connections_to_peers.items():
+            for src_peer, dst_peer in peer_pairs:
+                src_peer_name, _ = self._get_peer_name(src_peer, True)
+                if str(connections) == 'All connections' and src_peer == dst_peer:  # relevant with all connections only
```

Comment:
I think when you check if `connections` is all connections it is better to avoid string comparison, as the format might change. One option is to compare to `ConnectionSet(True)` object, another option - add a method `is_all()` to `ConnectionSet` indicating if the object allows all connections.

Reply:
Isn't it sufficient to only check the condition `src_peer == dst_peer` on this line?

Reply:
assuming that connections from peer to itself is always "All connections", the check `src_peer == dst_peer`  is sufficient